sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewTestSuite.scala

```diff
@@ -278,6 +279,19 @@ abstract class SQLViewTestSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("SPARK-34613: Fix view does not capture disable hint config") {
+    withSQLConf(DISABLE_HINTS.key -> "true") {
+      withView("v1") {
+        sql("CREATE VIEW v1 AS SELECT /*+ repartition(1) */ 1")
```

Comment:
let's follow other tests in this suite and use `createView`, which can test all 3 different view types.